ormconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "type": "mongodb",
+  "host": "",
+  "port": 0,
+  "username": "",
+  "password": "",
+  "database": "",
+  "synchronize": true,
```

Comment:
you should be able to leave these blank. Tbh I recommend switching to an ormconfig.js. Then you can do manipulation based on your NODE_ENV etc. Like we should probs add a docker-compose with mongo for local dev yea? So then based on prod vs dev vs testing we diff things with the variables

Synchronization is also not recommended in prod fyi.